### DISCUSSION TITLE
in chat(), Pressing Enter in the Chinese input method will send the message directly, which is not the case when using arg()

### DISCUSSION THREAD

**Discussion by alkene5** [johnlindquist/kit#1278] [Errors] [answered]
https://github.com/johnlindquist/kit/assets/61105068/35dcf4e0-0e4f-4d1f-81c3-0cf481b633ee

**Comment by johnlindquist** [comment]
Sorry, I don't understand what you're expecting... What happens when you use arg? I'm not familiar with the behaviors of the Chinese input method

**Reply by alkene5** [reply]
Chinese input method first input the pronunciation of Chinese characters, and then the candidate word appears, pressing the back car will put the candidate word into the input box, under normal circumstances, this carriage return will not be processed by other applications, but the chat() chat box feels that all carriage return behavior will be blocked, but the arg() function will not. They're listening for a different code on the return press?

**Reply by johnlindquist** [reply]
I did some research, it looks like I need to add events to detect when you open the Input Method Editor (IME). I'll add those for the next build this weekend, but I won't be able to test them.

When the next build is released, please test and let me know if it's fixed 🙏

**Reply by alkene5** [reply]
I will⭐

**Reply by alkene5** [reply, marked as answer]
It was fixed  1.76.4 😊 Thank you

**Reply by johnlindquist** [reply]
Awesome, I'm so glad this worked 🎉
